src/auth-service/config/emailConstants.js

```diff
@@ -0,0 +1,125 @@
+const emailConstants = {
+    greetings: (name) => {
+        return `<tr>
+                                <td
+                                    style="padding-bottom: 24px; color: #344054; font-size: 16px; font-family: Inter; font-weight: 600; line-height: 24px; word-wrap: break-word;">
```

Comment:
Hi @BenjaminSsempala , in a bid to minimise the number of new files, can't these be added to the default config object of the `constants` file?